components/org.wso2.carbon.identity.application.authentication.handler.session/src/main/java/org/wso2/carbon/identity/application/authentication/handler/session/internal/ActiveSessionsLimitHandlerServiceComponent.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.identity.application.authentication.handler.session.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.wso2.carbon.identity.application.authentication.framework.ApplicationAuthenticator;
+import org.wso2.carbon.identity.application.authentication.framework.UserSessionManagementService;
+import org.wso2.carbon.identity.application.authentication.handler.session.ActiveSessionsLimitHandler;
+import org.wso2.carbon.user.core.service.RealmService;
+
+/**
+ * OSGi declarative services component which handles registration and de-registration of Session Handler.
+ */
+@Component(
+        name = "identity.application.handler.session.component",
+        immediate = true
+)
+public class ActiveSessionsLimitHandlerServiceComponent {
+
+    private static final Log log = LogFactory.getLog(ActiveSessionsLimitHandlerServiceComponent.class);
+
+    @Activate
+    protected void activate(ComponentContext ctxt) {
+
+        try {
+            ActiveSessionsLimitHandler activeSessionsLimitHandler = new ActiveSessionsLimitHandler();
+            ctxt.getBundleContext().registerService(ApplicationAuthenticator.class.getName(), activeSessionsLimitHandler, null);
```

Comment:
Should be exceeding 120 lines count.

Reply:
Resolved in https://github.com/wso2-extensions/identity-local-auth-basicauth/pull/68/commits/aa9404e4bdae50bedd69f6a1492773da1a76bc80